samples/MvvmSampleUwp/Package.appxmanifest

```diff
@@ -33,19 +33,18 @@
       Executable="$targetnametoken$.exe"
       EntryPoint="MvvmSampleUwp.App">
       <uap:VisualElements
-        DisplayName="MVVM Toolkit Sample App"
-        Square150x150Logo="Assets\Square150x150Logo.png"
-        Square44x44Logo="Assets\Square44x44Logo.png"
-        Description="MVVM Toolkit Sample App"
-        BackgroundColor="#3750D1">
-        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png" Square71x71Logo="Assets\SmallTile.png" Square310x310Logo="Assets\LargeTile.png">
+        DisplayName="MVVM Toolkit Gallery"
+        Square150x150Logo="Assets\Icon\MedTile.png"
+        Square44x44Logo="Assets\Icon\AppList.png"
+        Description="MVVM Toolkit Gallery" BackgroundColor="transparent">
```

Comment:
```suggestion
        Description="MVVM Toolkit Sample App" BackgroundColor="transparent">
```

Reply:
WinUI uses Gallery, that's why I tweaked it. We can revert that.

".. App" feels a bit redundant though.. "MVVM Toolkit Samples" ?

Reply:
This is for consistency with the Windows Community Toolkit Sample App, plus it's also the name I already used to reserve the package identity in the Store. I think it's fine for a first release, we can always update it later on when we coordinate with the Windows Community Toolkit as well 🙂

Reply:
Fixed!